code/modules/tgui_panel/external.dm

```diff
@@ -16,8 +16,8 @@
 
 	nuke_chat()
 
-	// Failed to fix
-	action = alert(src, "Did that work?", "", "Yes", "No, switch to old ui")
+	// Failed to fix, using tgalert as fallback
+	action = tgalert(src, "Did that work?", "", "Yes", "No, switch to old ui")
```

Comment:
Keep such things as "fix verbs" using old alerts because even when everything breaks, these flows should still work.

Reply:
Just want to clarify - in the PR where tgalert has been replaced with tgui_alert, tgalert was kept as a fallback for the report issue button, so why should tgalert be replaced with alert here?

Reply:
Good question, maybe it is robust indeed.

Reply:
TGalert aint still html? It probably works as good as alert()